default/config/CommandDispatcherImplCfg.hpp

```diff
@@ -15,6 +15,12 @@ enum {
     CMD_DISPATCHER_SEQUENCER_TABLE_SIZE = 25, // !< The size of the table holding commands in progress
 };
 
-
+// Toggles whether events are triggered during command opcode registration. Disabling this can be
+// useful for memory-constrained systems with limited downlink buffer space as the events can crowd
+// out more useful events during initialization.
+#ifndef CMD_DISPATCHER_ENABLE_OPCODE_REGISTER_EVENTS
+#define CMD_DISPATCHER_ENABLE_OPCODE_REGISTER_EVENTS \
+    1  //!< Indicates whether or not command opcode registration events are triggered
+#endif
```

Comment:
What's the possibility of using a `constexpr bool` instead, and then using an ordinary `if` statement? That way, we can avoid more preprocessor conditionals and improve the typechecking, but a smart compiler can still eliminate the code.